src/main/java/com/nylas/Events.java

```diff
@@ -40,6 +40,11 @@ public long count(EventQuery query) throws IOException, RequestFailedException {
 	}
 	
 	public Event create(Event event, boolean notifyParticipants) throws IOException, RequestFailedException {
+		if(event.getConferencing() != null
+				&& event.getConferencing().getAutocreate() != null
+				&& event.getConferencing().getDetails() != null) {
+			throw new IllegalArgumentException("Cannot set both 'details' and 'autocreate' in conferencing object.");
+		}
```

Comment:
This covers the `create` case but not the `update` case.  Can an Event be updated to autocreate conferencing also?  What if it already has conferencing?

Reply:
Here are the new docs for the autocreate API: https://developer.nylas.com/docs/connectivity/calendar/autocreate-meetings. I have been able to test this and an example of how this would be used can be found in the PR's description.

Yes, you can update an event to autocreate if it does not have a conferencing object already present. However, if it already has conferencing, then you cannot add autoceate because it will throw an error on the server side.

PR has been updated to support validating the conferencing object for both creating and updating an Event.